keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/constraints/CanaryConstraintAttributes.kt

```diff
@@ -0,0 +1,7 @@
+package com.netflix.spinnaker.keel.api.constraints
```

Comment:
all of these are lifted from one file and shifted into their own files in the api module